utils/generate_types/index.js

```diff
@@ -396,8 +397,6 @@ function memberJSDOC(member, indent) {
   if (member.comment)
     lines.push(...member.comment.split('\n'));
   lines.push(...member.argsArray.map(arg => `@param ${arg.name.replace(/\./g, '')} ${arg.comment.replace('\n', ' ')}`));
-  if (member.returnComment)
```

Comment:
I'm curious where did this go?

Reply:
It is now always a part of the method body.